### ISSUE TITLE
All modules - configure maxIdle connections with sensible defaults 

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2202]
... or an instance can quickly consume 100 db connections, nearly all of them being in `idle` state:
![capture du 2018-10-10 11-11-24](https://user-images.githubusercontent.com/265319/46725839-5131da00-cc7d-11e8-8d58-e7807959276d.png)

**Comment by groldan** [comment]
Commit c3cead294dfdf1a63c1a5c0da2aedc7f0140c8c6 fixes it as part of #2414/PR #2436 

Manual test results:

- Just started with default settings (maxIdleTime: 60s):
```
georchestra=# select count(state), state from pg_stat_activity group by state;
 count | state  
-------+--------
     1 | active
    13 | idle
```

- `docker-compose stop geonetwork`:

```
 count | state  
-------+--------
     1 | active
     2 | idle
```

- access console and play around:
```
 count | state  
-------+--------
     1 | active
    10 | idle
```

- 60s later:
```
 count | state  
-------+--------
     1 | active
     4 | idle
```

- stop console: (has 2 conns minimum)
```
 count | state  
-------+--------
     1 | active
     4 | idle
```

- stop proxy: (has 1 conns minimum)
```
 count | state  
-------+--------
     1 | active
     4 | idle
```

- stop analytics: (has 1 conns minumum)
```
 count | state  
-------+--------
     0 | 
     1 | active
```

**Modify all apps .properties in ./config to set `dataSource.minPoolSize=0`**

- Just started:
```
georchestra=# select count(state), state from pg_stat_activity group by state;
 count | state  
-------+--------
     0 | 
     1 | active
    14 | idle
```

- 60s later:
```
 count | state  
-------+--------
     1 | active
    12 | idle
```

- stop geonetwork:
```
 count | state  
-------+--------
     1 | active
     1 | idle
```

- access console and play around a little:
```
 count | state  
-------+--------
     1 | active
    13 | idle
```

- 60s later:
```
 count | state  
-------+--------
     1 | active
     1 | idle
```

- stop console:
```
 count | state  
-------+--------
     0 | 
     1 | active
```

I couldn't make it go down to zero idle, and it's probably just some weird c3p0 thing that doesn't justify so much digging...

**Comment by groldan** [comment]
Closing, fixed by #2436
